File: app/src/main/java/com/softbankrobotics/qisdktutorials/ui/conversation/ConversationViewHolder.java
package com.softbankrobotics.qisdktutorials.ui.conversation;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.softbankrobotics.qisdktutorials.R;

/**
 * View holder for the conversation view.
 */
class ConversationViewHolder extends RecyclerView.ViewHolder {

    private final TextView textView;

    ConversationViewHolder(View itemView) {
        super(itemView);
        textView = itemView.findViewById(R.id.textView);
    }

    /**
     * Bind the text to the view.
     * @param text the text
     */
    void bind(String text) {
        textView.setText(text);
    }
}

File: app/src/main/java/com/softbankrobotics/qisdktutorials/ui/conversation/ConversationViewHolder.kt
/*
 * Copyright (C) 2018 Softbank Robotics Europe
 * See COPYING for the license
 */

package com.softbankrobotics.qisdktutorials.ui.conversation

import android.support.v7.widget.RecyclerView
import android.view.View

import kotlinx.android.synthetic.main.layout_info_log_view.view.*

/**
 * View holder for the conversation view.
 */
internal class ConversationViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    /**
     * Bind the text to the view.
     * @param text the text
     */
    fun bind(text: String) {
        itemView.textview.text = text
    }
}
